jboss-kie-common/added/launch/jboss-kie-common.sh

```diff
@@ -7,13 +7,13 @@ query_ocp_api() {
     local ocpApi="${1}"
     local resourcePath="${2}"
     # only execute the following lines if this container is running on OpenShift
-    if [ -e /var/run/secrets/kubernetes.io/serviceaccount/token ]; then
+    if [ -e /var/run/secrets/kubernetes.io/serviceaccount/token ] || [ "${K8S_ENV}" = true ] ; then
```

Comment:
Is this change required considering that token is created in bats test (just curious, I am not familiar with functionality of bats tests).

Reply:
You're right, I forgot to delete it, it was letf prior the previous tests I did.